ibm_catalog.json

```diff
@@ -17,138 +17,124 @@
-                            {
-                                "diagram": {
-                                    "caption": "Cluster security group rules example",
-                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/documentation/main/icons/ModuleExampleDiagram.svg",
-                                    "type": "image/svg+xml"
-                                },
-                                "description": "This example will add security rules to the `kube-<vpcid>` and `kube-<clusterId>` security groups."
-                            }
-                        ]
-                    }
-                },
-                {
-                    "label": "Cross account KMS encryption",
-                    "name": "cross_kms_support",
-                    "working_directory": "examples/cross_kms_support",
-                    "architecture": {
-                        "diagrams": [
-                            {
-                                "diagram": {
-                                    "caption": "Cross account KMS encryption example",
-                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/documentation/main/icons/ModuleExampleDiagram.svg",
-                                    "type": "image/svg+xml"
-                                },
-                                "description": "A single zone OCP VPC cluster configured with KMS encryption for cluster data and worker block storage using a KMS encryption key from another account."
-                            }
-                        ]
+                  "label": "quickstart",
+                  "name": "quickstart",
+                  "install_type": "fullstack",
+                  "working_directory": "solutions/quickstart-vpc",
+                  "configuration": [
+                    {
+                      "key": "ibmcloud_api_key"
+                    },
+                    {
+                      "key": "prefix",
+                      "required": true
+                    },
+                    {
+                      "key": "use_existing_resource_group"
+                    },
+                    {
+                      "key": "resource_group_name"
+                    },
+                    {
+                      "key": "cluster_name"
+                    },
+                    {
+                      "key": "ocp_version"
+                    },
+                    {
+                      "key": "number_worker_nodes"
+                    },
+                    {
+                      "key": "machine_type"
+                    },
+                    {
+                      "key": "vpc_id"
+                    },
+                    {
+                      "key": "existing_cos_id"
+                    },
+                    {
+                      "key": "region",
+                      "required": true,
+                      "options": [
```

Comment:
you dont need to use a custom dropdown here. You can use the out of the box VPC region picker since we only support VPC clusters